src/handlers/issue-matching.ts

```diff
@@ -41,44 +42,53 @@ export async function issueMatching(context: Context<"issues.opened" | "issues.e
   const issueContent = issue.body + issue.title;
   const commentStart = ">The following contributors may be suitable for this task:";
   const matchResultArray: Map<string, Array<string>> = new Map();
-  const similarIssues = await supabase.issue.findSimilarIssues(issueContent, context.config.jobMatchingThreshold, issue.node_id);
+  const similarIssues = await supabase.issue.findSimilarIssues({
+    markdown: issueContent,
+    threshold: context.config.jobMatchingThreshold,
+    currentId: issue.node_id,
+  });
   if (similarIssues && similarIssues.length > 0) {
-    similarIssues.sort((a, b) => b.similarity - a.similarity); // Sort by similarity
-    const fetchPromises = similarIssues.map(async (issue) => {
-      const issueObject: IssueGraphqlResponse = await context.octokit.graphql(
-        `query ($issueNodeId: ID!) {
+    similarIssues.sort((a: IssueSimilaritySearchResult, b: IssueSimilaritySearchResult) => b.similarity - a.similarity); // Sort by similarity
+    const fetchPromises = similarIssues.map(async (issue: IssueSimilaritySearchResult) => {
+      try {
+        const issueObject: IssueGraphqlResponse = await context.octokit.graphql(
```

Comment:
```suggestion
        const issueObject: IssueGraphqlResponse = await context.octokit.graphql(/* GraphQL */
```

Reply:
It should, if I didn't misspell it. Your IDE should understand the string is meant to be gql code and highlight / prettify.

Reply:
You should also probably add: https://github.com/ubiquity-os-marketplace/text-conversation-rewards/blob/4de373fc13fea8fc18c7d016379ad5b909321b87/graphql.config.yml